integration_tests/gdk/components/test_integ_PublishCommand.py

```diff
@@ -119,6 +119,108 @@ def test_publish_run_with_bucket_argument(mocker, get_service_clients, mock_proj
     assert spy_get_caller_identity.call_count == 1  # Get account number
 
 
+def test_publish_run_with_region_and_options_argument(mocker, get_service_clients, mock_project_config):
```

Comment:
Why are we testing region and options together? - Then why not include region, options and bucket

Reply:
This test actually has all three arguments. Will rename.